pos_reset_search/static/src/js/reset.js

```diff
@@ -1,12 +1,39 @@
 odoo.define("product_click_search_reset", function (require) {
```

Comment:
```suggestion
odoo.define("pos_reset_search.ProductScreen", function (require) {
```

Also rename file to `ProductScreen.js` please

Reply:
This one's still pending :)

We can merge after this is AFAICS

Reply:
This one's still pending @dsolanki-initos